### ISSUE TITLE
Add a customer tag to Find a Project page

### ISSUE THREAD

**Issue by anabasftp** [benwbrum/fromthepage#4140]
Shelter Island Historical Society emailed us recently that they couldn't find their institution in a search or anywhere on the dashboard. It's most likely because their collection is 100% transcribed, but we should still fix that and add a customer tag.

**Comment by saracarl** [comment]
I don't think this is quite a "customer" tag, as changes to the customer listing under the cloud tag in find a project.

**Comment by benwbrum** [comment]
The FindAProject page has been rewritten, and Staten Island Museum appears correctly now.
